lib/fog/compute/google/models/instance_group_manager.rb

```diff
@@ -0,0 +1,28 @@
+module Fog
+  module Compute
+    class Google
+      class InstanceGroupManager < Fog::Model
```

Comment:
This is missing a `.save` method.

Reply:
Done. Also added a bunch of attributes; reload and destroy.